projects/demo/src/modules/components/data-list-wrapper/examples/1/index.html

```diff
@@ -0,0 +1,16 @@
+<tui-combo-box
```

Comment:
Any examples? I just copied this one from a ComboBox page

Reply:
We need examples to demonstrate other inputs:
1. `disabledItemHandler`
2. `itemContent` (in particular `stringify | tuiStringifyContent` + `items | tuiFilterByInputWith : stringify`)
3. `labels`

Also `emptyContent` and `size`, but those can just be included in other more important examples.